corehq/apps/commtrack/models.py

```diff
@@ -193,6 +157,7 @@ class CommtrackConfig(QuickCachedDocumentMixin, Document):
     multiaction_enabled = BooleanProperty()
     multiaction_keyword_ = StringProperty()
 
+    # todo: remove?
     requisition_config = SchemaProperty(CommtrackRequisitionConfig)
     openlmis_config = SchemaProperty(OpenLMISConfig)
```

Comment:
Is OpenLMIS not a thing we need to support anymore? Just wanting to know
because I've come across other code that can be removed if that's the case